pkg/apb/client.go

```diff
@@ -175,20 +147,89 @@ func (c *Client) RunImage(
 	c.log.Debug("action:[ %s ]", action)
 	c.log.Debug("extra-vars:[ %s ]", extraVars)
 
+	// TODO(rhallisey): Cleanup the local vs incluster code paths so there
+	// is almost no variation between the two.
+	//
+	// If we're using a locally running broker we want to use the default
+	// namespace.
+	ns := context.Namespace
```

Comment:
I assume we'll want to keep line 155 since we reuse the 'ns' alias in a few spots.

I'd like add an explicit check for context.Namespace being sent and non-empty.
If we don't get a namespace in context.Namespace let's log a statement explicitly noting this and error out.